contracts/src/v0.8/functions/dev/v1_X/FunctionsBilling.sol

```diff
@@ -208,11 +208,20 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
       revert InsufficientBalance();
     }
 
-    bytes32 requestId = _computeRequestId(
-      address(this),
-      request.requestingContract,
-      request.subscriptionId,
-      request.initiatedRequests + 1
+    uint32 timeoutTimestamp = uint32(block.timestamp + config.requestTimeoutSeconds);
+    bytes32 requestId = keccak256(
+      abi.encode(
+        address(this),
+        request.requestingContract,
+        request.subscriptionId,
+        request.initiatedRequests + 1,
+        request.data,
+        request.dataVersion,
+        request.callbackGasLimit,
+        estimatedTotalCostJuels,
+        timeoutTimestamp,
+        tx.origin
+      )
```

Comment:
Explanation: we want to hash on everything that can change in a re-org:
- all inputs to Router.sendRequest()
- commitment fields that depend on block params (pretty much everything except static config)
- tx.origin to handle two EOAs submitting the exact same request (super unlikely but still)

Reply:
@justinkaseman @KuphJr does it make sense?